roles/build_dependencies/meta/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+galaxy_info:
+  role_name: build_dependencies
+  company: Princeton University Library
+  description: build_dependencies
+  author: pulibrary
```

Comment:
Should these be CDH? We could look at what we used before in the roles where we have added metadata - we should probably standardize if possible, although of course I want to give appropriate credit for PUL contributing to these ansible roles.